src/addrman.h

```diff
@@ -172,7 +172,7 @@ class CAddrInfo : public CAddress
 #define ADDRMAN_GETADDR_MAX_PCT 23
 
 //! the maximum number of nodes to return in a getaddr call
-#define ADDRMAN_GETADDR_MAX 2500
+#define ADDRMAN_GETADDR_MAX 1000
```

Comment:
When we're editing this file – what about skipping `#define` and using proper constants instead? :-)